third_party/houdini/lib/gusd/GEO_IOTranslator.cpp

```diff
@@ -71,7 +71,8 @@ checkExtension(const char* name)
     UT_String nameStr(name);
     if(nameStr.fileExtension()
      &&  (!strcmp(nameStr.fileExtension(), ".usd")
-       || !strcmp(nameStr.fileExtension(), ".usda"))) {
```

Comment:
Note we should check for something more robust once #532 is merged.

Reply:
Sorry, this was just a stop gap to cover a particular common case. Anything more general would definitely be better, thanks.